internal/cmd/root.go

```diff
@@ -443,21 +441,70 @@ func doStdInAndParse(ossIndex ossindex.IServer) (err error) {
 	}).Debug("Extracted purls")
 
 	logLady.Info("Auditing purls with OSS Index")
-	err = checkOSSIndex(ossIndex, purls, nil)
+	err = checkOSSIndex(ossIndex, mods, nil)
 
 	return err
 }
 
-func checkOSSIndex(ossIndex ossindex.IServer, purls []string, invalidpurls []string) (err error) {
-	var packageCount = len(purls)
-	coordinates, err := ossIndex.AuditPackages(purls)
+func checkOSSIndex(ossIndex ossindex.IServer, coordinates map[string]types.Projects, invalidpurls []string) (err error) {
+	var packageCount = len(coordinates)
+	purls := make([]string, 0, len(coordinates))
+	for k := range coordinates {
+		purls = append(purls, k)
+	}
+
+	ossIndexResponse, err := ossIndex.Audit(purls)
 	if err != nil {
 		return
 	}
 
+	// Wittle down list of audited to vulnerable stuff, so we can work faster
```

Comment:
typo: Whittle ;)

Reply:
WOOD YOU GIVE ME A BREAK!?!

Reply:
![](https://64.media.tumblr.com/a1bc8431d6e01e35d9a7c129e5306d3f/tumblr_mj21qawyWu1s5tlmxo3_500.gifv)